### DISCUSSION TITLE
【Arweave】Hackathon: Open Web Foundry

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#50] [Announcements]
Links: https://gitcoin.co/issue/arweaveteam/bounties/53/100027572
Time: Feb 2nd, 2022 - March 17th, 2022

Raise your hand to join the hackathon team!

**Comment by fewwwww** [comment]
🙋‍♂️

**Comment by Dream4ever** [comment]
🙋‍♂️

**Comment by kenspirit** [comment]
🙋‍♂️
